plugin.php

```diff
@@ -221,17 +221,13 @@ function create_initial_json_routes() {
 	 * Terms
 	 */
 	$controller = new WP_JSON_Terms_Controller;
+	$query_params = $controller->get_query_params();
 	register_json_route( 'wp', '/terms/(?P<taxonomy>[\w-]+)', array(
 		array(
-			'methods'  => WP_JSON_Server::READABLE,
-			'callback' => array( $controller, 'get_items' ),
-			'args'     => array(
-				'search'   => array(),
-				'per_page' => array(),
-				'page'     => array(),
-				'order'    => array(),
-				'orderby'  => array(),
-			),
+			'methods'       => WP_JSON_Server::READABLE,
+			'callback'      => array( $controller, 'get_items' ),
+			'args'          => $query_params,
+			'query_params'  => $query_params,
```

Comment:
It would be better if the server just knew where to find the schema and query params https://github.com/WP-API/WP-API/pull/949/files#r25561372